autoload/ale/handlers/eslint.vim

```diff
@@ -52,14 +52,20 @@ function! ale#handlers#eslint#GetCwd(buffer) abort
     let l:executable = ale#node#FindNearestExecutable(a:buffer, s:executables)
 
     if !empty(l:executable)
-        let l:nmi = strridx(l:executable, 'node_modules')
-        let l:project_dir = l:executable[0:l:nmi - 2]
```

Comment:
This line was causing the CWD to be set to `/path/to/project/eslint/bin/eslint.` instead of `/path/to/project` which causes the command to fail because `eslint/bin/eslint.` doesn't exist.